connection.go

```diff
@@ -93,3 +94,132 @@ func (e *EurekaConnection) UpdateApp(app *Application) {
 		}
 	}()
 }
+
+// AppUpdate is the outcome of an attempt to get a fresh snapshot of a Eureka
+// application's state, together with an error that may have occurred in that
+// attempt. If the Err field is nil, the App field will be non-nil.
+type AppUpdate struct {
+	App *Application
+	Err error
+}
+
+func sendAppUpdatesEvery(d time.Duration, produce func() AppUpdate, c chan<- AppUpdate, done <-chan struct{}) {
+	t := time.NewTicker(d)
+	defer t.Stop()
+	for {
+		select {
+		case <-done:
+			close(c)
+			return
+		case <-t.C:
+			// Drop attempted sends when the consumer hasn't received the last buffered update.
+			select {
+			case c <- produce():
+			default:
+			}
+		}
+	}
+}
+
+// ScheduleAppUpdates starts polling for updates to the Eureka application with
+// the given name, using the connection's configured polling interval as its
+// period. It sends the outcome of each update attempt to the returned channel,
+// and continues until the supplied done channel is either closed or has a value
+// available. Once done sending updates to the returned channel, it closes it.
+//
+// If prime is true, it sends at least one application update outcome to the
```

Comment:
I made the change to the `NewAppSource` method, forgetting to change this one too. I'll do that now.